ansible/group_vars/proxy-hosts.yml

```diff
@@ -261,7 +261,7 @@ nginx_proxy_direct_locations:
 - location: "^~ /mineotaur"
   redirect302: /about/mineotaur.html
 - location: "^~ /jupyter"
-  redirect302: https://idr-analysis.openmicroscopy.org
+  redirect302: /about/linked-resources.html
```

Comment:
Realized a bit too late. Should this have been

```diff
-  redirect302: https://idr-analysis.openmicroscopy.org
+  redirect302: /about/analysis-environments.html
```

instead?

Reply:
yes